SimpleColorApp/SimpleColorApp/ContentView.swift

```diff
@@ -8,17 +8,56 @@
 import SwiftUI
 
 struct ContentView: View {
-    var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundStyle(.tint)
-            Text("Hello, world!")
-        }
-        .padding()
+  
+  @State private var redSliderValue: Double
+  @State private var greenSliderValue: Double
+  @State private var blueSliderValue: Double
+  @State private var boxColor:Color
+  
+  init() {
+    let defaultRed = 200.0
+    let defaultGreen = 100.0
+    let defaultBlue = 240.0
+    
+    self.redSliderValue = defaultRed
+    self.greenSliderValue = defaultGreen
+    self.blueSliderValue = defaultBlue
+    self.boxColor = ContentView.calculateBoxColor(red: defaultRed, green: defaultGreen, blue: defaultBlue)
+  }
+  
+  var body: some View {
+    VStack {
+      Text("Color Picker")
+        .font(.largeTitle)
+      RoundedRectangle(cornerRadius: 0)
+        .fill(boxColor)
+      Text("Red")
+      HStack {
+        Slider(value: $redSliderValue, in: 1.0...255.0)
+        Text(String(Int(redSliderValue.rounded())))
+      }
+      Text("Green")
+      HStack {
+        Slider(value: $greenSliderValue, in: 1.0...255.0)
+        Text(String(Int(greenSliderValue.rounded())))
+      }
+      Text("Blue")
+      HStack {
+        Slider(value: $blueSliderValue, in: 1.0...255.0)
+        Text(String(Int(blueSliderValue.rounded())))
+      }
+      Button("Set Color") {
+        boxColor = ContentView.calculateBoxColor(red: redSliderValue, green: greenSliderValue, blue: blueSliderValue)
+      }
     }
+    .padding()
+  }
+  
+  private static func calculateBoxColor(red: Double, green: Double, blue: Double) -> Color {
```

Comment:
Adding `static` really doesn't feel right. But it's the best way to meet the requirements. Most people either make this part `static` or else make some or all of the variables at the beginning `static`. I'm bringing this up because you might get the impression that you're going to be writing `static` all of the time, and you're not. It has very limited uses. The keyword says "here is a function or variable that you can use without making an instance of this class/struct" We usually use these for convenience methods or common constants we want to use throughout the app.